build.gradle

```diff
@@ -380,6 +430,7 @@ plugins.withType(JavaPlugin) {
   tasks.javadoc.options.links "http://typesafehub.github.io/config/latest/api/",
                               "https://docs.oracle.com/javase/7/docs/api/",
                               "http://docs.guava-libraries.googlecode.com/git-history/v15.0/javadoc/"
+  rootProject.ext.javadocLinks.each{tasks.javadoc.options.links it}
```

Comment:
Is there no programmatic way to fetch these for each library we use? Or do we have to manually add them?

If its manual it would probably be good to add the Hadoop javadocs (some other nice ones would be Avro,